### ISSUE TITLE
Does adminer:dg support mongo?

### ISSUE THREAD

**Issue by bagasabisena** [dockette/adminer#16]
It is mentioned in the README.md that adminer with `dg` tag (`adminer:dg`) supports Mongo.

But when I look the Dockerfile, there is no related package for MongoDB, as shown below

```bash
RUN echo '@community http://nl.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories && \
    echo '@testing http://nl.alpinelinux.org/alpine/edge/testing' >> /etc/apk/repositories && \
    apk update && apk upgrade && \
    apk add \
        wget \
        ca-certificates \
        php7@community \
        php7-session@community \
        php7-mysqli@community \
        php7-pgsql@community \
        php7-json@community && \
    wget https://github.com/dg/adminer-custom/archive/v$ADMINER_DG_VERION.tar.gz -O /srv/adminer.tgz && \
    tar zxvf /srv/adminer.tgz --strip-components=1 -C /srv && \
    rm /srv/adminer.tgz && \
    apk del wget ca-certificates && \
    rm -rf /var/cache/apk/*
```

whereas in the php7 full tag, there is MongoDB related package in the Dockerfile

```bash
RUN echo '@community http://nl.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories && \
    echo '@testing http://nl.alpinelinux.org/alpine/edge/testing' >> /etc/apk/repositories && \
    apk update && apk upgrade && \
    apk add \
        wget \
        ca-certificates \
        php7@community \
        php7-session@community \
        php7-mysqli@community \
        php7-pgsql@community \
        php7-mongodb@testing && \
        wget https://github.com/vrana/adminer/releases/download/v$ADMINER_VERSION/adminer-$ADMINER_VERSION.php -O /srv/index.php && \
    apk del wget ca-certificates && \
    rm -rf /var/cache/apk/*
```

So, does `adminer:dg` supports mongoDB? Or should I use the full one?

BTW, thank you for your good work with this project! It is really helpful.

**Comment by bagasabisena** [comment]
Alright, I'll try my best 👍

**Comment by f3l1x** [comment]
ping @bagasabisena

**Comment by f3l1x** [comment]
ping @bagasabisena

**Comment by enumag** [comment]
I believe this is solved by #23 and can be closed.

**Comment by f3l1x** [comment]
Right. Thanks @enumag
